client/client_test.go

```diff
@@ -193,32 +195,52 @@ func TestAddLeaf(t *testing.T) {
 	cli := trillian.NewTrillianLogClient(env.ClientConn)
 	for _, test := range []struct {
 		desc    string
+		leaf    []byte
 		client  trillian.TrillianLogClient
 		wantErr bool
```

Comment:
`wantErr` unused -- preferably add an error test or two (e.g. use an invalid logID) rather than removing the field